contracts/modules/TransactionManager.sol

```diff
@@ -91,15 +95,13 @@ abstract contract TransactionManager is BaseModule {
      * @param _wallet The target wallet.
      * @param _target The address to add.
      */
-    function addToWhitelist(
-        address _wallet,
-        address _target
-    )
-        external
-        onlyWalletOwnerOrSelf(_wallet)
-        onlyWhenUnlocked(_wallet)
+    function addToWhitelist(address _wallet, address _target) external
+    onlyWalletOwnerOrSelf(_wallet)
+    onlyWhenUnlocked(_wallet)
     {
-        require(!isWhitelisted(_wallet, _target), "TT: target already whitelisted");
+        require(_target != _wallet, "TM: Cannot whitelist wallet");
+        require(!IWallet(_wallet).authorised(_target), "TM: Cannot whitelist module");
```

Comment:
Shouldn't we just do `require(_target != address(this))` to save some gas since there is only 1 module?

Reply:
To help us make a decision the measured difference in gas is 4,580 (88,404 vs 83,824).